modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-tracker/src/main/java/com/liferay/portal/osgi/web/portlet/tracker/internal/PortletTracker.java

```diff
@@ -1308,6 +1308,8 @@ private PortletApp _createBundlePortletApp(
 
 		BundleContext bundleContext = bundle.getBundleContext();
 
+		_servletContextHelperFactory.registerServletContextHelperRegistration();
```

Comment:
@MrJohn1911 Not really helping to postpone these to actual usage, because these methods are called during startup (for example `WabFactory` is a self starting bundle tracker). I think it's better to just put the calls in their activate methods.